python/cugraph/cugraph/community/ktruss_subgraph.py

```diff
@@ -159,7 +160,7 @@ def ktruss_subgraph(G, k, use_weights=True):
     _ensure_compatible_cuda_version()
 
     KTrussSubgraph = Graph()
-    if type(G) is not Graph:
+    if G.is_directed():
         raise Exception("input graph must be undirected")
```

Comment:
It would be good to clean these up when we see them: the exception type for these checks should be `ValueError`.